### ISSUE TITLE
Mobile data problem in Realme c1

### ISSUE THREAD

**Issue by custosoft** [phhusson/treble_experimentations#2194]
Hello. I hope you can read this. Always disconnect and reconnect the data even if the ims feature is on. I hope that you fix this. 

Phone: Realme C1
ROM: AOSP 12 v402

![Screenshot_20220212-222744](https://user-images.githubusercontent.com/74230345/153716271-cc56dc9d-1c31-4c8f-8372-b7bb6ca1c37e.png)

![Screenshot_20220212-225923](https://user-images.githubusercontent.com/74230345/153716389-0aa26839-0bfb-4e40-a925-0632568e4e6c.png)

Here is my logcat.
[2022-02-12-18-15-44.txt](https://github.com/phhusson/treble_experimentations/files/8053700/2022-02-12-18-15-44.txt)

**Comment by phhusson** [comment]
This means that when you reboot, mobile data does work right? It shuts down only later?

**Comment by custosoft** [comment]
This is a weird thing about that issue. Still got intermediately disconnected and reconnect. My mobile data is turned on.

https://user-images.githubusercontent.com/74230345/154611813-0a64a3af-15a2-4ee6-982e-aa0bd8dd8f94.mp4

**Comment by custosoft** [comment]
and my answer is no. Even reboot or shut down. this issue is not fixed

**Comment by custosoft** [comment]
Yes when ims feature is disable. When i turn off, the LTE icon is disappeared and data is totally disconnected. And yeah. my phone got have a problem because of:

My Data hasn't enough data balance.
VPN
Low memory. my phone is 2GB
Vendor problem. All files in vendor are untouched.

**Comment by custosoft** [comment]
Follow up: even turned on data and when i browse, watch video and play online games, the LTE icon disappeared and appeared again.
